PWGCF/JCorran/Core/JFFlucAnalysis.h

```diff
@@ -113,6 +133,7 @@ class JFFlucAnalysis
       fh_vertex[iaxis]->Fill(fVertex[iaxis]);
   };
 
+#define NK nKL // for cpplint nonsense
```

Comment:
Please, for the next iteration, no emotions in the comments

Reply:
Alright 😅